docs/Lua API.rst

```diff
@@ -1004,6 +1004,22 @@ Fortress mode
   Resets the fortress mode sidebar menus and cursors to their default state. If
   ``pause`` is true, also pauses the game.
 
+* ``dfhack.gui.pauseRecenter(pos[,pause])``
+  ``dfhack.gui.pauseRecenter(x,y,z[,pause])``
+
+  Same as ``resetDwarfmodeView``, but also recenter if ``x`` isn't ``-30000``,
+  and respects RECENTER_INTERFACE_SHUTDOWN_MS in DF's init.txt.
```

Comment:
What does the `RECENTER_INTERFACE_SHUTDOWN_MS` setting do? I'm asking because I don't know, but you should probably explain in the docs because others might not know too

Reply:
DF's init.txt says:
"This controls the number of milliseconds that must pass before input works again after the view recenters on an event in dwarf mode."
Not sure why we'd need to document DF's features, but I guess it wouldn't hurt.

Reply:
A link to existing df docs would be fine too